Sources/MapboxNavigation/Route.swift

```diff
@@ -70,17 +70,15 @@ extension Route {
                     var feature = Feature(geometry: .lineString(LineString(congestionSegment.0)))
                     feature.properties = [
                         CongestionAttribute: String(describing: congestionSegment.1),
-                        "isAlternativeRoute": isAlternativeRoute,
```

Comment:
Note to self: this is no longer necessary because we’re putting the alternative route in a separate source to be rendered by a separate layer, no longer using data-driven styling to consolidate layers and sources.